typings/index.d.ts

```diff
@@ -140,10 +131,23 @@ declare module 'mappersmith' {
     gatewayConfigs: GatewayConfiguration
     maxMiddlewareStackExecutionAllowed: number
     middleware: Middleware[]
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
     Promise: Promise<any>
```

Comment:
You didn't change this, but this doesn't seem right. I think this should be `PromiseConstructor`, no?

Reply:
🧑‍⚖️  Agreed, it should.